website/project/views/node.py

```diff
@@ -832,6 +833,17 @@ def _view_project(node, auth, primary=False,
             for key, value in settings.NODE_CATEGORY_MAP.iteritems()
         ]
     }
+
+    region_list = get_storage_region_list(user)
+
+    # Default should be at top of list for UI and for the project overview page the default region
+    # for a component is that of the it's parent node.
+    if user:
+        default_storage_region = {'name': node.osfstorage_region.name, '_id': node.osfstorage_region._id}
```

Comment:
The `osfstorage_region` property makes a query to find the node's storage region:
https://github.com/CenterForOpenScience/osf.io/blob/63dc0c1fc73f3c4464dcab22c273387f2bdef9bd/osf/models/node.py#L992-L996

Instead of calling it three times in this function, how about storing the region as a variable so you can access the `name` and `_id` without performing additional queries?